src/ToDo.UI/App.xaml.cs

```diff
@@ -118,19 +113,19 @@ protected async override void OnLaunched(LaunchActivatedEventArgs args)
 #endif
 #endif
 
-		if (Host.Services.GetService<IRouteNotifier>() is { } notif)
+		var host = BuildAppHost();
```

Comment:
App host should be the first thing that's built - I'm ok with the refactoring to a separate BuildAppHost method but it needs to be done before the App is initialized to ensure that things like logging and config are available before any other resources for the app are loaded. 
I get that this is so that the resourceloader is available for the viewmaps but for that I think we need an alternative that defers resource resolution to the point where the messagedialog is requested.

Reply:
For the moment, let's approve these changes and just note that we need a better long term solution for localizing strings